grammars/python-console.cson

```diff
@@ -0,0 +1,17 @@
+'scopeName': 'text.python.console'
+'name': 'Python Console'
+'fileTypes': [
+  'pycon'
+]
+'patterns': [
+  {
+    'match': '^(>{3}|\\.{3}|In \\[\\d+\\]:) (.+)$'
+    'captures':
+      '1':
+        'name': 'punctuation.separator.prompt.python-session'
```

Comment:
I think this should be `punctuation.separator.prompt.python.console` to match the grammar's scope name.